plugin.go

```diff
@@ -232,42 +137,104 @@ func rsyncPages(workspacePath string, pagesDirectory string, temporaryBaseDirect
 		"--exclude",
 		"CNAME",
 		"-r",
-		pagesDirectory,
-		temporaryBaseDirectory,
+		p.Config.PagesDirectory,
+		p.Config.TemporaryBase,
 	)
-	cmd.Dir = workspacePath
-	return cmd
+
+	cmd.Dir = p.Build.Path
+	return runCommand(cmd)
 }
 
-// Add the files in the temporary directory to the commit
-// we want to make in the clone.
-func addTemporaryFilesToClone(temporaryPagesDirectory string) *exec.Cmd {
+func (p Plugin) stageChanges() error {
 	cmd := exec.Command(
 		"git",
 		"add",
-		"--verbose",
```

Comment:
Should we keep a verbose output?

Reply:
The dirtyRepo already prints the modified or new files, I dropped the verbose flag because it also just prints the files that get added as this had been redundant.

Reply:
Cool that sounds good to me